catalogue/templates/catalogue/includes/component-card.html

```diff
@@ -38,6 +38,18 @@
             Learn More
           </a>
         </div>
+        <button
+          id="addComponent"
+          class="button is-light"
+          style="position: absolute; right: 0; bottom: 0;"
+          hx-post="{% url 'catalogue:add_cart' c.id %}"
+          hx-target="#shopping_card"
+          hx-swap="innerHTML"
+        >
```

Comment:
should not be shown on the frontpage. maybe add some bool to hide the button? Default would be show...